src/main/java/ToDo.java

```diff
@@ -0,0 +1,22 @@
+/**
+ * Todo class, subclass of Task
+ */
+public class ToDo extends Task {
+
+    /**
+     * Constructor for Todo class
+     * @param description
+     */
+    public ToDo(String description){
```

Comment:
Should there be a space after the (
```suggestion
    public ToDo(String description) {
```